### DISCUSSION TITLE
Move note

### DISCUSSION THREAD

**Discussion by lghio** [tamlok/vnote#2311] [Ideas]
It would be nice to be able to move a note from one folder to another.
Today, it's too cumbersome, I need to unlink the note, move the file using the file explorer then reload the new folder to be taken into account by Vnote.

**Comment by tamlok** [comment]
Why not try the context menu via one single right mouse button click on the node explorer?

**Reply by lghio** [reply]
There is no Move option

**Reply by tamlok** [reply]
> There is no Move option

There is a Copy/Cut/Paste option.
